lms/djangoapps/teams/csv.py

```diff
@@ -19,3 +28,220 @@ def load_team_membership_csv(course, response):
         "Team membership CSV download is not yet implemented."
     )
     response.write(not_implemented_message + "\n")
+
+
+class TeamMemberShipImportManager(object):
+    """
+    A manager class that is responsible the import process of csv file including validation and creation of
+    team_courseteam and teams_courseteammembership objects.
+    """
+
+    def __init__(self):
```

Comment:
Suggestion: take the course as a parameter to your `__init__` - we shouldn't have one of these objects ever operate on two or more different courses.